source/openapi3/split.yaml

```diff
@@ -4508,6 +4513,7 @@ components:
       example:
         data:
           id: 55afddde-4296-4daf-8e49-7ba481ef9608
+          ref: oustanding-tours
```

Comment:
Contact refs start with a CNT (unfortunately 😂 ) so it should be `CNT.123`

Reply:
Updated: 44e08a2